tests/countries/test_french_southern_territories.py

```
@@ -33,17 +33,17 @@ def test_no_holidays(self):
         self.assertNoHolidays(FrenchSouthernTerritories(years=1955))
 
     def test_2022(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)